framework/src/media/audio/audio_manager.c

```diff
@@ -980,7 +980,7 @@ int start_audio_stream_in(void *data, unsigned int frames)
 	return ret;
 }
 
-int start_audio_stream_out(void *data, unsigned int frames)
+int start_audio_stream_out(stream_info_t *stream_info, void *data, unsigned int frames)
```

Comment:
Stream info is created when application started.
It will be called at once media player created, or will be called by specific reason.
Why don't you re-use set_stream_policy? (please see comment regarding set_stream_policy)
Then you don't have to add stream_info as a first parameter of each function, card_info already knows it.